interfaces/apparmor/backend.go

```diff
@@ -395,15 +396,24 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	for i, profile := range changed {
 		pathnames[i] = filepath.Join(dir, profile)
 	}
-	errReloadChanged := loadProfiles(pathnames, cache, skipReadCache)
+
+	var errReloadChanged error
+	timings.Run(tm, "load-profiles", fmt.Sprintf("load changed security profiles of snap %q", snapInfo.InstanceName()), func(nesttm timings.Measurer) {
+		errReloadChanged = loadProfiles(pathnames, cache, skipReadCache)
```

Comment:
I wonder if we should distinguish this one from the one below?

"load-profiles[changed]"  vs "load-profiles[unchanged]"  perhaps ?

Reply:
Good idea, done.